src/components/Enterprise/UniversalSection.tsx

```diff
@@ -1,20 +1,29 @@
 import { FunctionComponent } from 'react'
 
+import Link from 'next/link'
+
 import { ContentSection } from '..'
 
 import { DevPlatformsSection } from './DevPlatformsSection'
 
 export const UniversalSection: FunctionComponent = () => (
     <ContentSection
-        className="flex max-w-[1232px] flex-col justify-between overflow-hidden  rounded-2xl  border-1 border-gray-200 bg-white px-6 md:flex-row md:px-16"
-        parentClassName="md:px-[80px] !py-0"
+        className="flex flex-col justify-between overflow-hidden rounded-2xl border-1 border-gray-200 bg-white py-16 px-6 md:flex-row md:py-0 md:px-16"
+        parentClassName="md:px-20 !py-0"
     >
-        <div className="flex flex-col justify-center pt-16 !pb-12 md:w-[492px] md:py-16">
-            <img alt="language models brand" className="mb-6 w-12" src="/enterprise/language-models-brand-icon.svg" />
-            <h1 className="mb-4">Universal</h1>
-            <p className="mb-0 text-[24px] leading-[30px] -tracking-[0.25px]">
-                You don't have to compromise. Sourcegraph works with all major code hosts and many of the most popular
-                IDEs.
+        <div className="mb-[57px] flex flex-col justify-center md:mb-0 md:max-w-[560px] md:py-[120px]">
+            <h2 className="mb-4">Works with your existing code hosts and IDEs</h2>
+            <p className="mb-0 text-xl leading-[26px] -tracking-[0.25px] text-gray-500">
+                Cody lives in VS Code and JetBrains IDEs and works with code from any code host.
+            </p>
+            <p className="mt-4 text-xl leading-[26px] -tracking-[0.25px]  text-gray-500">
+                <Link
+                    className="!px-0 !py-0  leading-[22px] text-violet-500 underline underline-offset-4"
+                    href="/enterprise"
+                >
+                    Cody Enterprise
+                </Link>{' '}
```

Comment:
Please remove violet coloring from these inline links. They should just be the default body color used for the surrounding text, but with an underline treatment.

Reply:
Certainly, we'll remove the violet from the inline links and apply the default text color with underline and font weight as detailed in the style guide.
Thank you